problem_builder/tests/integration/test_step_builder.py

```diff
@@ -379,6 +379,38 @@ def test_html_last(self):
 
         self.html_section(step_builder, controls, last=True)
 
+    def test_step_builder_contains_only_one_step_at_a_time(self):
+        """
+        Check step builder contains only a single step at a time.
+
+        Step builder steps are detached from the DOM (and stored in javascript) shortly after they are loaded,
+        Only the currently visible step is attached to the DOM. This tests checks that when we are interacting with
+        the Step Builder filling the questions there is only a single step in the DOM.
+        """
+        step_builder, controls = self.load_assessment_scenario(
+            "step_builder.xml",
+            {"max_attempts": 0, "extended_feedback": False}
+        )
+
+        def single_step_present_in_dom():
+            self.assertEqual(len(step_builder.find_elements_by_css_selector('.sb-step')), 1)
+
+        def no_steps_present_in_dom():
+            self.assertEqual(len(step_builder.find_elements_by_css_selector('.sb-step')), 0)
```

Comment:
@jbzdak Thanks for adding these functions. Can you add an `assert_` prefix to these two functions to indicate that that's what they do?